R/metadata.R

```diff
@@ -14,11 +14,16 @@ cache <- rlang::env(
 #' @return A character scalar consisting of the URL
 #' @examples
 #' get_metadata(remote_url = DATABASE_URL)
```

Comment:
You need to update the documentation for this function. It doesn't have to be comprehensive, because it's just an internal